docs(api): liquid classes in 8.5 (#18619)
OpenTrons/opentrons-api: api/docs/v2/new_examples.rst

Advanced Method
---------------

> These protocols accomplishes the same thing as the previous example, but does it a little more efficiently. Notice how it uses the :py:meth:`.InstrumentContext.transfer` or :py:meth:`.InstrumentContext.transfer_with_liquid_class` method to move liquid between well plates. Because each is a complex command, you don't need separate calls to ``aspirate`` or ``dispense`` here.

Reviewer: ```suggestion
These protocols accomplish the same thing as the previous example, but a little more efficiently. Notice how they use the :py:meth:`.InstrumentContext.transfer` or :py:meth:`.InstrumentContext.transfer_with_liquid_class` methods to move liquid between well plates. Because each is a complex command, you don't need separate calls to ``aspirate`` or ``dispense`` here.
```

                p300.transfer(100, plate["A1"], plate["B1"])

> When you use the liquid class command ``transfer_with_liquid_class()``, you'll need to specify a ``liquid_class`` along with volume, source and destination, and trash parameters. 

Reviewer: `trash_location` is optional and i think we should omit it in this example, which is trying to be as simple as possible.

> Opentrons-verified liquid class definitions are based on Flex pipette and tip combinations. The API will raise an error if you try to perform a liquid class transfer with an OT-2 pipette and tips. 

Reviewer: since technically you can build your own OT-2 liquid class:
```suggestion
Opentrons-verified liquid class definitions are based on Flex pipette and tip combinations. The API will raise an error if you try to perform a liquid class transfer with one of these definitions in an OT-2 protocol. 
```

.. tabs::

    .. tab:: Flex

        .. code-block:: python
            :substitutions:

            from opentrons import protocol_api

>             metadata = {"apiLevel": "|apiLevel|"}

Reviewer: this needs to be a `requirements` dictionary and specify Flex

            def run(protocol: protocol_api.ProtocolContext):
                plate = protocol.load_labware(
                    load_name="corning_96_wellplate_360ul_flat",
                    location="D1"
                )
                tiprack_1 = protocol.load_labware(
                    load_name="opentrons_flex_96_tiprack_200ul",
                    location="C1"
                )

>                 p50 = protocol.load_instrument(

Reviewer: Using this pipette causes an error. We haven't defined liquid classes for the larger tip sizes (here 200) on the smaller pipettes. Use a 1000 instead.
